ui.frontend/src/csrfRequest.js

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import {FunctionRuntime, request} from '@aemforms/af-core'
+
+export const enableCSRF = () => {
+    FunctionRuntime.registerFunctions(window.Granite.csrf.request(request))
```

Comment:
we need to update af-core library first before merging this. else all runtime test cases will fail.